blog/2017-08-10-RealTimePlanningUsingKieServerPart1.adoc

```diff
@@ -0,0 +1,149 @@
+= Optimize your problems using KIE Server, part 1
+:page-interpolate: true
+:awestruct-author: mcimbora
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, howto, kie server]
+
+KIE Server provides an execution environment separated from your client application.
+Run it on the same local network as your client, or in the cloud, depending on your needs.
```

Comment:
The first few lines in a blog post are very important - they need to make the reader want to read the rest of it. They need to make him curious. "how can this help me?" Also the first 3 lines or so appear in the blog overview.

These 2 lines do not make me want to read this article (sorry for being so blunt). They're technical implementation details. Things that can come later if the article. The first paragraph should explain:
- value proposition
- Purpose
- Proof / examples
See also "PPP" in this video: https://www.youtube.com/watch?v=aJFK9k5i1ew